server/components/form-profile/form-profile.html

```diff
@@ -47,28 +47,31 @@
       }) }}
 
       {% call radioWrapper({
-        label: 'And your gender?',
+        label: 'Gender',
         hook: 'gender-form'
       }) %}
 
         {{ radio({
           name: 'gender',
           value: 'Male',
           image: '/assets/images/male.png',
+          authentication: isAuthenticated,
```

Comment:
What does the IsAuthenticated do? Just curious

Reply:
check whether it's your own profile, to show a form to change your profile